src/pages/developers/tbtc-security-model.md

```diff
@@ -0,0 +1,126 @@
+
+While tBTC’s governance has been designed to be restrictive and resilient in the face of malicious developers or a stolen key, as with any project making claims to censorship-resistance, the ongoing role of the development team and all code deserve heavy scrutiny by users and outside parties.
+
+## Additional attacks
+
+The strongest attack known against the tBTC sidechain mechanism requires collusion of three roles -- the price feed maintainer, a large number of signers, and new depositors.
+
+* First, the feed maintainer manipulates the reported prices such that ETH is, for example, 100% overvalued relative to BTC. This manipulation needs to be maintained over time.
+* New depositors can now see that any BTC they deposit will now only have 75% recourse in case of fraud, and should avoid the system. Colluding depositors, however, continue to deposit BTC.
+* Honest signers will continue to custody BTC, regardless of the recourse collateralization. If there are a large number of malicious signers in the signing set, however, this is an opportunity to collude and abscond with BTC in low-collateral deposits. These signers will be liquidated, losing 75% of 1 BTC for every BTC they steal, as well as being ejected en masse from the network for future signing.
+* For each 1 BTC removed from the system, either 75% of 1 BTC in ETH is auctioned off or the same remains available for auction; any deposit whose auction has not been taken is breaking the supply peg until such a time as TBTC value is worth the auctioned bond.
+
+This attack is difficult, requiring significant capital and coordination. The strongest mitigation [is Keep’s random beacon](https://blog.keep.network/whats-in-a-beacon-12c34b0bc078), requiring a large portion of malicious signers across the staked candidate pool, or extraordinary luck on the side of the attacker.
+
+## Alternative security models (BTCB, RenBTC)
+
+There are many alternate models for pegged Bitcoin, each with varying levels of security. Centralized Bitcoin banks like [Binance’s BTCB](https://www.binance.com/en/blog/347360878904684544/Introducing-BitcoinPegged-Token-on-Binance-Chain) offer a simple solution - they handle the pegging of BTC, but you have to trust them to do so. Permissioned federations of companies such as Liquid and [Wrapped Bitcoin](https://www.wbtc.network/) are the same model with different stakeholders.
```

Comment:
"but you have to trust them to do so. "

maybe something like

"but you have to trust them to do so. While things have come along way since MtGox, human systems still rely on trusting humans."

Reply:
Too much salt for this doc, though agreed in principle